packages/data-broker/src/__tests__/services/dhis/pullers/EventsPuller.test.js

```diff
@@ -86,13 +95,107 @@ describe('EventsPuller', () => {
     it('translates data source element to DHIS element codes if required', () =>
       assertEventAnalyticsApiWasInvokedCorrectly({
         dataSources: [DATA_GROUPS.POP01_GROUP],
-        options: { dataElementCodes: ['POP01', 'DIF01'] },
+        options: { dataElementCodes: ['POP01', 'DIF01'], organisationUnitCodes: ['DL_FACILITY_A'] },
         invocationArgs: expect.objectContaining({ dataElementCodes: ['POP01', 'DIF01_DHIS'] }),
       }));
   });
 
   describe('data building', () => {
+    describe('pulling events', () => {
```

Comment:
Minor: I think `data building` means just the building part of it rather than pulling as well, maybe pop `pulling events` outside of this?